server/src/main/java/com/iqas/server/service/DocumentService.java

```diff
@@ -0,0 +1,52 @@
+package com.iqas.server.service;
+
+
+import com.iqas.server.model.DocumentModel;
+import com.iqas.server.model.KbModel;
+import com.iqas.server.repository.DocumentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class DocumentService {
+
+    @Autowired
+    private DocumentRepository documentRepository;
+
+    @Autowired
+    private NextSequenceService nextSequenceService;
+
+    @Autowired
+    private KbService kbService;
+
+    private List<DocumentModel> getAllDocuments() {
+        return documentRepository.findAll();
+    }
+
+    public DocumentModel addDocument(DocumentModel documentModel, Integer kbId) {
+
+        KbModel retKbModel = kbService.findOneKb(kbId);
+        DocumentModel retModel =null;
+        if (retKbModel == null) {
+            retModel= null;
+        } else {
+            documentModel.setDocId(nextSequenceService.getNextSequence("customSequencesDoc"));
+            DocumentModel savedDocument = documentRepository.save(documentModel);
+            List<Integer> docIdListIni = null;
+            if (retKbModel.getDocId() == null)
```

Comment:
This confused me a little. Maybe we should call the property `docIds` and the method `getDocIds()` to make it explicit that it is a collection of document IDs?

Reply:
https://github.com/divyanshbajpai/iqas/pull/3/commits/4887a40eb48d0ac80f5271c79a60c36c2d160dcb